common/config.go

```diff
@@ -99,7 +99,7 @@ func StagenetConfig() *Config {
 				Port: 38081,
 			},
 		},
-		SwapCreatorAddr: ethcommon.HexToAddress("0xbf2B7a6dCE5598Cf002B3507a8D62cf2C35cE5c6"),
+		SwapCreatorAddr: ethcommon.HexToAddress("0xAD031B747757d5d308e0f28544AF162A89782B6E"),
```

Comment:
we need to update mainnet too i suppose?

Reply:
Yes, when there are no more changes to make, we'll update it. I made the `t.Skip(...)` in `TestMainnetContract` loud so that we don't forget.